<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="w_bit">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="title">
     <string>Word Size</string>
    </property>
    <widget class="QRadioButton" name="w_16bit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>15</y>
       <width>60</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>16 bits</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="w_32bit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>15</y>
       <width>60</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>32 bits</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="w_64bit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>15</y>
       <width>60</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>64 bits</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="RoundCount_Box">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>60</y>
      <width>145</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string>Round (0-32000)</string>
    </property>
    <widget class="QSpinBox" name="RoungCount">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>115</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>32000</number>
     </property>
     <property name="value">
      <number>20</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="PassLenght_Box">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>145</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string>Pass Len (1-255)</string>
    </property>
    <widget class="QSpinBox" name="PassLength">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>115</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>16</number>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>190</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>250</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>120</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
